corehq/apps/accounting/subscription_changes.py

```diff
@@ -191,6 +193,21 @@ def response_domain_security(domain):
             domain.secure_sessions = False
             domain.save()
 
+    @staticmethod
+    def response_report_builder(project):
+        from corehq.apps.userreports.models import ReportConfiguration
+        reports = ReportConfiguration.by_domain(project.name)
+        builder_reports = filter(lambda report: report.report_meta.created_by_builder, reports)
+        for report in builder_reports:
+            try:
+                report.config.deactivate()
+                # It's theoretically possible that someone could have a
+                # hand-built report based on one of these data sources,
+                # in which case they will be sad.
```

Comment:
I'm just going to remove this comment since this isn't really a workflow we imagine people should be using, and the failure mode isn't terrible, they just will need to kick off a table rebuild (which they will know how to do if they are writing UCRs "by hand")